client/src/document/on-github.tsx

```diff
@@ -124,7 +124,7 @@ function NewIssueOnGitHubLink({ doc }: { doc: Doc }) {
   } else {
     url.pathname = "/mdn/content/issues/new";
     sp.set("template", "page-report.yml");
-    sp.set("mdn-url", doc.mdn_url);
+    sp.set("mdn-url", `https://developer.mozilla.org${doc.mdn_url}`);
```

Comment:
Might be nice to have a comment here noting that `doc.mdn_url` isn't actually a URL so nobody else gets stung by this 🙄 😆 

```suggestion
    sp.set("mdn-url", `https://developer.mozilla.org${doc.mdn_url}`); // doc.mdn_url is actually a slug
```